web-common/src/features/entity-management/resource-status-utils.ts

```diff
@@ -106,12 +106,17 @@ export function resourceStatusStore(
       )
         return { status: ResourceStatus.Busy };
 
+      const changed =
+        !lastUpdatedOn ||
+        (res.data?.meta?.stateUpdatedOn
+          ? res.data?.meta?.stateUpdatedOn > lastUpdatedOn
+          : undefined);
+
```

Comment:
Not sure if we want to use `undefined` here. @ericpgreen2 would have more context.

Reply:
I'm not so familiar with this code (more @AdityaHegde), but it looks like it's fine as long as we protect against accessing an `undefined` attribute. And it looks like @bcolloran is doing so on line 154. After a quick look, line 154-155 is the only place where I see this `changed` attribute being used.

Reply:
good catch @djbarnwal, thank you. The current code is 
`!lastUpdatedOn || res.data?.meta?.stateUpdatedOn > lastUpdatedOn`, and `res.data?.meta?.stateUpdatedOn` is `string | undefined`. Since `undefined > x  => false` for all `x: string`, we should default to false here.

Reply:
(but that said we also return undefined for `changed` from this function on line 107 right above this: `return { status: ResourceStatus.Busy }`)